SQL.js

```diff
@@ -81,6 +85,10 @@ class SqlStatement {
         data = data.transform()
         quote = ''
       }
+      if (data instanceof SqlStatement) {
```

Comment:
nit: use else if to avoid checking this condition if the earlier condition matched